content/blog/jjpx/2018-09-05_haiku_monthly_activity_report_august_2018.md

```diff
@@ -0,0 +1,43 @@
++++
+type = "blog"
+title = "Haiku monthly activity report - 08/2018"
+author = "jjpx"
+date = "2018-09-05 16:37:40-03:00"
+tags = []
++++
+
+Time has come and a new monthly activity report arrived with it. It is the first time I write something this long in english. Enjoy!
+
+This report covers hrev52141-52310.
+
+### Drivers
+
+SDHCI MMC driver from [krish_iyer](https://www.haiku-os.org/blog/krish_iyer) has been merged in master. There is still a lot of work to do and it is not enabled in automated builds yet, but should serve as a basis for any future related work. His commits include extensive documentation on how to setup a testing vm, what has been done and what is left to do.
+
+There have been several contributions to the ARM port including work by kallisti5 on our implementation fo Flattened Device Tree (FDT) that should make it easier for Haiku to be run on more ARM hardware variants and a rewrite of the memcpy function for ARM devices by Augustin Cavalier based on Clang 8.
+
+Finally on the driver related news kallisti5 fixed an SMAP violation on s3 accelerant.
+
+### Fixes
+
+PulkoMandy fixed a bug in MediaPlayer related to playing streams of unknown length such as an online radio.
+
+phuodoin fixed a bug where the system became unresponsive when a textsearch is used in a big search.
+
+### Others
+
+waddlesplash fixed and enabled Reiserfs on x86_64 builds. Previously this filesystem was only available on x86 builds.
+
+Zstandard has been added as an alternative and high ratio compression algorithm for Haiku Packages. Although it is not the default, a quick test by diver reduced LibreOffice package from 112MB to 107MB with best compression.
+
+Thanks to 3dEyes who [enabled SubtleCrypto in QtWebKit](https://discuss.haiku-os.org/t/whatsapp-web/7165/13), **WhatsApp Web** is now available on Qt based browsers (Qupzilla, Otter and more).
+
+New in HaikuDepot: **UploadIt** is a Tracker Add-On that uploads a single file to the online service The Null Pointer at https://0x0.st 5 . The URL for the uploaded file is put into the clipboard after the upload has finished, ready to be pasted into an email, a chat window or forum post.
+
+### Beta is closer than ever! (I can _feel it_)
+
+Thanks to some intense work done by waddlesplash we have now a R1 Beta 1 [release timeline](https://dev.haiku-os.org/wiki/R1/Beta1/Timeline). Note that they are not strict deadlines to favor a high quality release instead of a rushed one.
```

Comment:
While waddlesplash is the release coordinator and has indeed put a lot of work into Haiku lately, this is also made possible by the cooperation of several others in the Haiku team. It would be nice to credit everyone for their hard work in the past 6 years.